defaults/main.yml

```diff
@@ -20,3 +19,9 @@ mesos_zookeeper_masters: "zk://{{ zookeeper_hostnames }}/mesos"
 # Containerizer
 mesos_containerizers: "mesos"
 mesos_executor_timeout: "5mins"
+
+# Additional configurations
+mesos_configs: []
```

Comment:
Using list or dict for default variable is tricky because the variable gets entirely overridden, i.e. both declaration don't get their values merged: highest precedence will overwrite the variable. Meaning we can't provide predefined configs in there without having the user redefining them if he wants to add more.

My suggestion would be to rename it to `mesos_additional_configs` to make it clear these are not the main defaults configs and that is up to the user to provide any value. Then we should never add default value in the array to avoid the merge issue.